.gitignore

```diff
@@ -12,3 +12,7 @@
 hs_err_pid*
 DataManager/src/Tests/
 Serverp/client_log.dat
+Serverp/version.dat
+/nbproject/private/
+/AndroidAppTests/nbproject/private/
```

Comment:
Is there any need to include AndroidAppTests as we are no longer needing unit tests? Or does it need renaming.